pkgs/top-level/all-packages.nix

```diff
@@ -1688,7 +1688,9 @@ in
 
   b3sum = callPackage ../tools/security/b3sum {};
 
-  backblaze-b2 = callPackage ../development/tools/backblaze-b2 { };
+  backblaze-b2 = callPackage ../development/tools/backblaze-b2 {
+    setuptools_scm = python3Packages.callPackage ../development/python-modules/setuptools_scm/2.nix { };
```

Comment:
Why do we need version 2? Version 3 should work if we widen the version constraint.

Reply:
You're right. Fixed